packages/core/src/CorePlugin.ts

```diff
@@ -61,6 +61,77 @@ export class CorePlugin extends JWPlugin {
             handler: this.toggleList.bind(this),
         },
     };
+    defaultShortcuts = [
+        //
+        // personal computer
+        //
```

Comment:
What is the point of this comment ? Explaining what `pc` means as a `platform` ?